protocol/0012-POSR-position_resolution.md

```diff
@@ -22,147 +22,45 @@ Any trader that has insufficient collateral to cover their margin liability is r
-- Entire distressed position should always be liquidated - even if reducing position size, by say 50%, would result in the remaining portion being above the trader's maintenance margin.
-- When there's insufficient volume on the order-book to close out a distressed position no action should be taken: the position remains open and any amounts in trader's margin account should stay there. Same principle should apply if upon next margin recalculation the position is still distressed.
-- If the party is distressed at a point of leaving auction it should be closedout immediately (provided there's enough volume on the book once all the pegged and liquidity provision orders get redeployed).
-
-## Examples and Pseudo code
+- If a party marked as "distressed" has the cross marginging mode selected for the market being considered:
 
-### _Scenario -  All steps_
-
-`Trader1 open position: +5`
-`Trader1 open orders:  0`
-`Trader2 open position: -4`
-`Trader2 open orders:   0`
-`Trader3 open position: +2`
-`Trader3 open orders:   0`
-
-#### STEP 1
+  - all their open orders on that market get cancelled without releasing any collateral from the margin account,
+  - their margin level then gets recalculated to account only for the open volume they hold in a market
+  - if party's margin account balance is now above or equal to their latest maintenance margin level then such party is no longer marked as "distressed".
 
-No traders are removed from the distressed trader list.
+- If party's is marked as "distressed" at this stage then:
 
-#### STEP 2
+  - their open volume get's added to network's open volume for that market,
+  - party's open volume in that market gets set to 0,
+  - the full amount of party's margin account balance gets transferred to market's insurance pool.
 
-`NetOutstandingLiability = 5 - 4 + 2 = 3`
+This concludes the position resolution from party's perspective.
 
-#### STEP 3
+#### Managing network's position
 
-```json
-LiquiditySourcingOrder: {
-  type: 'market',
-  direction: 'sell',
-  size: 3
-}
+Whilst network has a non-zero position in a given market it's treated as any other party with market's insurance in that market acting as its margin account and with and exception that margin search, margin release or liquidation are never attempted on a network party.
 
-LiquiditySourcingTrade1: {
-  buyer: Trader4,
-  seller: Network,
-  size: 2,
-  price: 120,
-  type: 'liquidity-sourcing'
-}
+Whenever the network party has a non-zero position it attempts to unload it using an [immediate or cancel](./0014-ORDT-order_types.md) limit order. If the the network has a `long` position it will submit a `sell` order. If it has a short position it will submit a buy order. The size of that order is chosen according to the liquidation strategy which forms a part of the marke's configuration. The strategy can be updated at any point whilst market is active with a market change [governance vote](./0028-GOVE-governance.md#2-change-market-parameters).
```

Comment:
I suspect this should be an IOC _market_ order, rather than a limit order

Reply:
The idea was that the limit order would be safer for the network as it'd more difficult to manipulate things to that it trades at some extremely unfavourable price